docs/manuals/source/Appendix/HardwareSizing.rst

```diff
@@ -36,31 +39,41 @@ Depending on the number of files and the average length of filenames, the **data
 
 To **calculate the number of files** in the DB, the number of files being backed up from all systems needs to be multiplied by the number of times that they will be kept in the database.
 
-The **amount of data per file** in the DB, depends on the size of filenames that are being backed up, but we have analyzed some real-world examples and found that values between **250 and 350 bytes per row** are usual.
+The **amount of data per file or path** in the DB, depends on the size of filenames or path length that are being backed up, but we have analyzed some real-world examples and found that values between **250 and 350 bytes per row** are usual.
```

Comment:
Is that true? Is that also the value for path entries? Usually I would expect the path entries to be much longer than the file entries?

Reply:
Check real customer report in task OP #5209 section file